src/N98/Magento/Command/System/Cron/RunCommand.php

```diff
@@ -94,6 +94,9 @@ protected function execute(InputInterface $input, OutputInterface $output)
                     ->setFinishedAt(strftime('%Y-%m-%d %H:%M:%S', time()))
                     ->save();
             } catch (Exception $e) {
+                $output->writeln('');
```

Comment:
There is the \Symfony\Component\Console\Application::renderException method, see src/N98/Magento/Application.php:602 for an example, I think it only needs little adoption for a command.